ui/src/components/Header.js

```diff
@@ -1,45 +1,59 @@
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
+import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import history from '../history';
 import '../styles/header.css';
 import { logoutUser } from '../redux/actions';
+import gameService from '../services/gameService';
+import DropdownSearch from './DropdownSearch';
 
 const Header = ({ siteTitle }) => {
-  const [searchTerm, setSearchTerm] = useState('');
   const dispatch = useDispatch();
   const isLoggedIn = useSelector((state) => state.auth.loggedIn);
+  const suggestionsMapper = async (searchTerm) => {
+    const response = await gameService.getSuggestions({ searchTerm });
+    if (response.status !== 200) {
+      return [];
+    }
+    const newSuggestions = response.body;
+    return newSuggestions.map((suggestion) => (
+      <Link
+        to={`/group/${suggestion.id}`}
+        className="suggestion-container"
+        key={suggestion.id}
+      >
+        <div className="suggestion">
+          <img alt={suggestion.thumbUrl} src={suggestion.thumbUrl}/>
+          <div className="suggestiontitle">
+            {suggestion.name}
+          </div>
+        </div>
+      </Link>
+    ));
```

Comment:
It feels like the jsx should be in DropdownSearch.js. Perhaps the callback returns the suggestions and you map them to components in that file instead?

Reply:
Fixed this, thanks